log/slog.go

```diff
@@ -187,7 +187,7 @@ func (rl *RelayLogger) WithModule(
 	}
 }
 
-func (rl *RelayLogger) TimeTrack(start time.Time, name string) {
+func (rl *RelayLogger) TimeTrack(start time.Time, name string, otherArgs ...any) {
 	elapsed := time.Since(start)
-	rl.Logger.Info("time track", "name", name, "elapsed", elapsed.Nanoseconds())
+	rl.Logger.Info("time track", "name", name, "elapsed", elapsed.Nanoseconds(), otherArgs)
```

Comment:
You pass a slice to a function as variadic arguments, you need to append `...` after the slice.

Reply:
Certainly `... ` otherwise it would not be formatted correctly.
https://github.com/hyperledger-labs/yui-relayer/pull/117/commits/5ab866778cc7af7f59c96bf37070fa53cadaecdf
Fixed